sdk/spring/spring-integration-azure-core/src/main/java/com/azure/spring/integration/instrumentation/DefaultInstrumentationManager.java

```diff
@@ -15,6 +15,11 @@
 
     private final Map<String, Instrumentation> healthInstrumentations = new ConcurrentHashMap<>();
 
+    /**
+     * Get health instrumentations.
+     *
+     * @return healthInstrumentations the health instrumentations
+     */
```

Comment:
please refactor the funtion name as "getAllHealthInstrumentation", same for the java doc.